enterprise/admin/__init__.py

```diff
@@ -410,3 +410,39 @@ def preview_as_program(self, request, obj):  # pylint: disable=unused-argument
     preview_as_program.short_description = _(
         "Preview the HTML template rendered in the context of a program enrollment."
     )
+
+
+@admin.register(EnterpriseCourseEnrollment)
+class EnterpriseCourseEnrollmentAdmin(admin.ModelAdmin):
+    """
+    Django admin model for EnterpriseCourseEnrollment
+    """
+
+    class Meta(object):
+        model = EnterpriseCourseEnrollment
+
+    list_display = (
+        'enterprise_customer_user',
+        'course_id',
+        'consent_granted'
+    )
+
+    search_fields = ('enterprise_customer_user__user_id', 'course_id',)
+
+
+@admin.register(PendingEnrollment)
+class PendingEnrollmentAdmin(admin.ModelAdmin):
+    """
+    Django admin model for PendingEnrollment
+    """
+
+    class Meta(object):
+        model = PendingEnrollment
+
+    list_display = (
+        'user',
+        'course_id',
+        'course_mode',
+    )
+
+    search_fields = ('course_id',)
```

Comment:
Would you also want to be able to search on a user or enterprise customer attribute here?